hydra-node/test/Hydra/Chain/Direct/ContractSpec.hs

```diff
@@ -112,6 +114,9 @@ spec = parallel $ do
       propTransactionEvaluates healthyCollectComTx
     prop "does not survive random adversarial mutations" $
       propMutation healthyCollectComTx genCollectComMutation
+  describe "Decrement" $ do
+    prop "is healthy" $
+      propTransactionEvaluates healthyDecrementTx
```

Comment:
are we adding mutation specs for this in a future PR?

Reply:
Yes. This should be right after we have the on-chain code since mutations test the on-chain part.